backend/Backend.mo

```diff
@@ -33,6 +33,14 @@ actor class Backend(_owner : Principal) {
   var userTokens = HashMap.HashMap<Principal, Types.UserToken>(5, Principal.equal, Principal.hash);
   var developerUsers = Buffer.Buffer<Types.Developer>(10);
   var developerPendingApproval = Buffer.Buffer<Types.DeveloperApproval>(10);
+
+  var creatorUsers = Buffer.Buffer<Types.Creator>(10);
+  var creatorPendingApproval = Buffer.Buffer<Types.CreatorApproval>(10);
```

Comment:
change this to only have a single variable creatorUsers and figure out pending users using status refer how developerTools have been done